src/ucp/rma/amo_send.c

```diff
@@ -140,7 +140,7 @@ ucs_status_ptr_t ucp_atomic_fetch_nb(ucp_ep_h ep, ucp_atomic_fetch_op_t opcode,
         goto out;
     }
 
-    req = ucp_request_get(ep->worker);
+    req = ucp_request_get(ep->worker, "ucp_atomic_fetch_nb");
```

Comment:
maybe remove "ucp_" from all names, since it's all upc requests

Reply:
> maybe remove "ucp_" from all names, since it's all upc requests

ok, removed "ucp_" prefix